### ISSUE TITLE
Add list of all no-UI options

### ISSUE THREAD

**Issue by ArkadiuszMichalski** [notepad-plus-plus/npp-usermanual#94]
Please add list with all hidden options that are not available directly in the UI. I'm sure I've seen other options somewhere else, but I can't locate them. A separate list with such hiding settings would be very helpful.

[From DonHo](https://community.notepad-plus-plus.org/topic/19514/7-8-7-release-candidate-4/2?_=1591365029113&lang=en-US):
```
<FindHistory nbMaxFindHistoryPath="10" nbMaxFindHistoryFilter="10" nbMaxFindHistoryFind="10" nbMaxFindHistoryReplace="10" matchWord="no" matchCase="no" wrap="yes" directionDown="yes" fifRecuisive="yes" fifInHiddenFolder="no" dlgAlwaysVisible="no" fifFilterFollowsDoc="no" fifFolderFollowsDoc="no" searchMode="0" transparencyMode="1" transparency="150" dotMatchesNewline="no" isSearch2ButtonsMode="yes" "regexBackward4PowerUser"="yes">
Add an non-UI option to allow regEx backward search
Set regexBackward4PowerUser to yes in config.xml to eanable this option.
```

[From Michael Vincent](https://community.notepad-plus-plus.org/topic/19512/backward-regular-expression-searching-disabled/8?_=1591364679037):
```
<GUIConfig name="commandLineInterpreter">powershell</GUIConfig>
Which makes the menu File => Open Containing Folder => cmd actually open PowerShell instead of cmd.exe.
```

**Comment by pryrt** [comment]
per [this post[(https://community.notepad-plus-plus.org/post/54710), Don has volunteered for this one.  Thanks!

**Comment by pryrt** [comment]
closed by #95
